updater/src/Updater/Generate/Changelog.purs

```diff
@@ -100,9 +102,12 @@ fetchNextPageOfReleases page gh = do
   case result of
     Left err -> do
       throwError (error $ AX.printError err)
-    Right { body } ->
+    Right { status } | status == (StatusCode 404) -> do
+      pure Nothing
+    Right { body, status } ->
```

Comment:
We should update this to verify that the status code is 200. On anything else, we should throw an error. If you page PureScript for its third page of releases where each page has the maximum 100 per page, you will still get a 200. As proof, see this url: https://api.github.com/repos/purescript/purescript/releases?per_page=100&page=3